keras_nlp/models/whisper/whisper_encoder.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 The KerasNLP Authors
```

Comment:
Should we add a test file file with one quick unit tests to assert that there is no bias in some way? Might be nice to get a test failure that will warn in case something changes upstream that breaks our method.

Reply:
Good point

Reply:
I've added a UT in `whisper_backbone_test.py`